modules/homa-lite/README.md

```diff
@@ -0,0 +1,83 @@
+# Homa Lite Module
+
+## Overview
+The Homa Lite module allows users to upload some Staking currency, and mint some Liquid currency.
+
+### User Functions
+Users can:
+* Request Mint: Request to upload Staking currency.  
+* Claim: After a batch has been processed, the user can claim the Liquid currency minted from the Staking currency uploaded.
+
+### Root Functions 
```

Comment:
```suggestion
### Governance origin dispatchable calls
```

The homa-lite module itself doesn't make assumption what the origin would actually be. The descriptions below need to change as well.